meson.build

```diff
@@ -322,7 +322,8 @@ features = {
     'extra_deps': [ pahomqttc_dep ]
   },
   'tvm-support': {
-    'extra_deps': [ tvm_dep, dlpack_dep ]
+    'extra_deps': [ tvm_dep, dlpack_dep ],
+    'project_args': {'ALLOC_ALIGNMENT': 1}
```

Comment:
Because tvm subplugin may be built independently (any subplugin may be built independently), let's not change core behaviors with a subplugin availability.

Instead, you may add "enable-alloc-alignment" in meson_option, make it default "TRUE".
And generate a warning or an error at build of tvm-support if alloc-alignment is FALSE.

Reply:
You may need to look at api.git for allocations as well. (Single API)

Reply:
Sounds right! Updated